test/check-storage-reclaim

```diff
@@ -208,6 +208,79 @@ class TestReclaim(VirtInstallMachineCase, StorageCase):
 
         r.check_disk_row("vda", parent="vda2", size="1.00 GB", action="resized from 4.29 GB")
 
+    def testExtendedPartitionEFI(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        s = Storage(b, m)
+        r = Review(b, m)
+
+        # Configure disk layout with a logical Btrfs partition
+        disk = "/dev/vda"
+        s.partition_disk(
+            disk,
+            [
+                ("512MB", "efi"),
+                ("2GB", "swap"),
+                ("8GB", "ext4"),
+                ("", "extended"),
+                ("2.5GB", "logical"),
+            ],
+            is_efi=True
+        )
+
+        # Create subvolumes for the logical Btrfs partition
+        m.execute(f"""
+        mkfs.btrfs -f -L BTRFS {disk}5
+        mount {disk}5 /mnt
+        btrfs subvolume create /mnt/@subvol1
+        btrfs subvolume create /mnt/@subvol2
+        umount /mnt
+        """)
+
+        s.udevadm_settle()
+
+        # Open the installer and navigate to the reclaim space screen
+        i.open()
+        i.reach(i.steps.INSTALLATION_METHOD)
+        s.rescan_disks()
+
+        # Configure disk partitioning with reclaim space
+        s.set_partitioning("use-free-space")
+        s.reclaim_set_checkbox(True)
+        i.next(True)
+
+        # **Update: Verify the partition layout in the user interface**
```

Comment:
Is the **Update word here left by mistake?

Reply:
Exactly, I'll fix it